uniffi_bindgen/src/interface/types/mod.rs

```diff
@@ -67,6 +67,9 @@ pub enum Type {
     External { name: String, crate_name: String },
     // Custom type on the scaffolding side
     Custom { name: String, builtin: Box<Type> },
+    // An unresolved type inside a proc-macro exported function signature.
+    // Must be replaced by another type before bindings generation.
```

Comment:
I think it's worth mentioning that this will only be a `Record`/`Object`/`Enum`, etc. and not a primitive type like `u8` or `String`.  I don't know it's an official UniFFI term, but I've been calling these "user types".

Reply:
I've updated it to "user-defined type".